docs/source/index.rst

```diff
@@ -77,6 +77,7 @@ Features described in this documentation are classified by release status:
    torch.distributed.algorithms.join <distributed.algorithms.join>
    torch.distributed.elastic <distributed.elastic>
    torch.distributed.fsdp <fsdp>
+   torch.distributed.xeon <distributed.xeon>
```

Comment:
is `torch.distributed.xeon` an actual sub-package?

Reply:
Or are you actually documenting this launcher script?
torch/backends/xeon/run_cpu.py
Cc: @kurman @d4l3k

Reply:
Discussed this offline. This has been implemented as a xeon based launcher and we should make this a torchrun handler and special args can be passed as rdzv-conf. This will simplify UX and unify features.
As a doc PR, looks good to me at the high level.

Reply:
Yes, we document torch.backends.xeon launcher script. We will rename the doc.